src/python/WMCore/ReqMgr/Service/RequestAdditionalInfo.py

```diff
@@ -57,6 +57,23 @@ def create_web_splitting_format(split_info):
     return web_form
 
 
+def create_updatable_splitting_format(split_info):
+    splitInfo = []
+    for taskInfo in split_info:
+        if taskInfo["taskType"] not in ["Cleanup", "LogCollect"]:
+            splittingTask = taskInfo["taskName"]
```

Comment:
It seems we can remove `splittingTask`. Can you also add a short docstring to this method?